src/SourceBuild/patches/runtime/0001-Also-pass-NETCoreSdkRuntimeIdentifier-to-runtime.pro.patch

```diff
@@ -0,0 +1,21 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Adeel Mujahid <3840695+am11@users.noreply.github.com>
+Date: Wed, 26 Jun 2024 11:04:39 +0300
+Subject: [PATCH] Also pass NETCoreSdkRuntimeIdentifier to runtime.proj
+
+---
+ eng/Subsets.props | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/eng/Subsets.props b/eng/Subsets.props
+index 68569344a6e..532928f67da 100644
```

Comment:
I think this index is wrong, we need to create patch from the dotnet/dotnet to get the correct index: https://github.com/dotnet/source-build/blob/main/Documentation/patching-guidelines.md (I can't get the eng/vmr-sync.sh to succeed locally just remembered how @ViktorHofer does the patching 🙂 )

Reply:
Hmm I followed that doc and it doesn't say you need to do it from dotnet/dotnet...

Reply:
My understanding is this is about runtime project under dotnet/dotnet tree:

> To create a repo patch file, first commit your changes to the source repo (e.g. runtime, aspnetcore) as normal,

but I could be wrong. Index mismatch is the only thing which I could make sense of. Maybe we need to checkout `d0c8f836b5d46cd7c24fb7b7a8292b45d97f8e42`

```sh
cd runtime
git checkout d0c8f836b5d46cd7c24fb7b7a8292b45d97f8e42
curl -sSL https://github.com/dotnet/runtime/pull/104012.patch | git apply -
git format-patch --zero-commit --no-signature -1 
```

Reply:
Just tried it and it produces the same indices. So it's a mystery.

Reply:
Yes, you can create the patch from outside of dotnet/dotnet.

Reply:
akoeplinger reported this to me and it's a bug in the sync process which I am fixing now. There's a bug when adding a patch for currently changed lines of code. Should be fixed today.

Reply:
Thanks. I think we can revert this patch once https://github.com/dotnet/runtime/pull/104012 is merged.

Reply:
Sorry, I was unable to merge the fix today. I will try to do it tomorrow.

Reply:
Ok, I have a PR with the fix: https://github.com/dotnet/arcade-services/pull/3690
It turned out to be quite a nasty bug.

Reply:
Thanks, we have reverted this patch as it has flowed through the runtime. You fix will definitely help in future endeavors. 👌